components/core/src/clp_s/ArchiveWriter.hpp

```diff
@@ -74,9 +135,22 @@ class ArchiveWriter {
      */
     void initialize_schema_writer(SchemaWriter* writer, Schema const& schema);
 
-    size_t m_encoded_message_size;
+    /**
+     * Stores the tables
+     * @return Size of the compressed data in bytes
+     */
+    size_t store_tables();
+
+    /**
+     * Updates the archive's metadata
```

Comment:
Probably needs more detail -- I can't tell whether this is talking about local archive metadata (e.g. timestamp dict, schemas, etc.) or global metadata db